packages/docs/docs/miscellaneous/local-development.md

```diff
@@ -4,96 +4,168 @@ sidebar: auto
 
-VuePress is using a combo with [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) and [Lerna](https://github.com/lerna/lerna).
+## Prerequisites
 
-## Init packages
+- [Node.js v12.x](https://nodejs.org/en/)\*
+- [Yarn v1.x](https://classic.yarnpkg.com/)
+
+\*Node.js v10 should work as well, but other versions have not been verified.
+
+## Setup Guide
+
+In this guide, we will be using the following names to refer to the two projects you need to be successful:
+
+- **VuePress Project**: This refers to your fork of the [official VuePress repo](https://github.com/vuejs/vuepress/)
+- **VuePress Sandbox**: This refers to a local instance of VuePress that will serve as the simulation for creating test scenarios to verify that changes in the VuePress Project work as expected
+
+### VuePress Project Setup
+
+1. Fork the [official VuePress repo](https://github.com/vuejs/vuepress/)
+1. Clone your fork onto your machine (e.g., `git clone ...`)
+1. Open your cloned project in a new terminal window
+1. Run the following commands:
 
 ```bash
- yarn // it will install dependencies of all packages
+# Install all dependencies in the project
+yarn
+
+# Compile shared-utils TypeScript package into JavaScript
+yarn tsc
 ```
 
-`yarn` will use hoisting. What does it mean for you ?
+5. Verify there is no global installation of VuePress
 
-It will regroup all dependencies in the workspace root and link all packages.
+```bash
+# Check global yarn packages
+yarn global list
 
-Check the link by running the following command:
+# If it exists, remove global VuePress package
+yarn global remove vuepress
+```
+
+6. Configure local VuePress Project to be the source of truth
 
 ```bash
-    ls -la node_modules/@vuepress
+# Registers local VuePress project
+yarn register-vuepress
+
+# If successful, you should see a message
+# like `success Registered "vuepress"`
 ```
 
-:::warning
-You have to take care to declare all dependencies inside subFolders package.json. When publish the lib if dependencies from a package is not declare it will just not work.
-:::
+And with that, we're ready to setup our VuePress Sandbox!
 
-:::warning
-There is a special package you should have a look is @vuepress/shared-utils that are in typescript.
-:::
+### VuePress Sandbox Setup
 
-After install everything it will run `yarn tsc`. This command will tell to @vuepress/shared-utils workspace to compile his js.
+1. In a separate terminal, create a new npm project.
 
-:::warning
-From here if you are making change inside this package you will have to
-run `yarn tsc` all the time or run in separate shell `yarn run tsc -w`. This will re run tsc at any change from shared-utils
-:::
+```bash
+# Create a new folder
+mkdir vuepress-sandbox
 
-## Link
+# Change directory to VuePress Sandbox
+cd vuepress-sandbox
 
-Good from here you have everything ready. You need to link VuePress to your project.
+# Initalize a new npm project
+npm init
```

Comment:
it would be nice to mention  `yarn init` as an alternative too :)

Reply:
Good suggestion! Just added it